superset-frontend/src/explore/actions/exploreActions.ts

```diff
@@ -152,6 +153,7 @@ export const exploreActions = {
   updateChartTitle,
   createNewSlice,
   sliceUpdated,
+  updateDataMask,
```

Comment:
Small point to consider - JavaScript has allowed the notion of trailing commas in objects since ES5 and it simplifies the ```git diff``` on a file since it will only highlight a single line when adding k_,v_ pairs. Trailing commas however are disallowed in JSON since it's based off of a syntax prior to ES5. Let's figure out as a team how consistent we want to be going forward and potentially create a set of lint rules we can universally apply to every PR.